vision/rectangle.py

```diff
@@ -0,0 +1,7 @@
+class Rectangle:
```

Comment:
Module docstring at top. Class docstring right under definition -- describe parameters of init there too.